src/components/UpdatesFeed/index.js

```diff
@@ -0,0 +1,51 @@
+import { useMemo } from "react";
+import { makeStyles } from "tss-react/mui";
+
+import useRecentActivity from "queries/useRecentActivity";
+
+import UpdateCard from "./UpdateCard";
+import UpdatesHeader from "./UpdatesHeader";
+
+const useStyles = makeStyles()((theme) => ({
+	container: {
+		display: "flex",
+		flexDirection: "column",
+		gap: "10px",
+		padding: "1rem 1rem 1rem 3rem",
+		flex: "0 0 66%",
+	},
+}));
+
+const UpdatesFeed = ({ setSelectedTab }) => {
+	const handleTabButtonClick = (index) => {
+		setSelectedTab(index);
+	};
+
+	const { classes } = useStyles();
+
+	const { data = [] } = useRecentActivity();
+	const updates = useMemo(
+		() =>
+			data.map(({ _id, content, dateUpdated, headline, documentUrl, project, pcp }) => ({
+				key: _id,
+				updateContent: content,
+				updateDate: dateUpdated,
+				updateName: headline,
+				project,
+				documentUrl,
+				pcp,
+			})),
+		[data],
+	);
+
+	return (
+		<div className={classes.container}>
+			<UpdatesHeader onButtonNavClick={handleTabButtonClick} />
+			{updates.map((update) => (
```

Comment:
For accessibility, we will want this to be rendered as a list. We can tackle the accessibility issues later, but I thought I would mention it now, as a reminder that we need to take accessibility into consideration.

Reply:
Not sure exactly what you mean by list here, as html list `<li>`? Leaving as is for now to address with other accessibility stuff :).